### PR TITLE
Added new plugin flyte [flyte.org]

### PR DESCRIPTION
**Pull request by evalsocket**
- [x] Make sure you read the Plugin Naming Guide: https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/
- [x] Verify you can install your plugin locally

Below is the test output:
```
Installing plugin: flyte
Installed plugin: flyte
\
 | Use this plugin:
 |      kubectl flyte
 | Documentation:
 |      https://github.com/lyft/flyte
/

```

### CODE REVIEW

File: plugins/flyte.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: flyte
+spec:
+  version: "v0.5.12"
+  homepage: https://github.com/lyft/flyte
+  shortDescription: kubectl-flyte is an command line tool for flyte that can be used as an extension to kubectl.
```
Comment (ahmetb) [inline plugins/flyte.yaml:8]: "Kubectl-flyte is a command line tool" is redundant. Keep it short and descriptive. 

Right now this description does not really convey any meaning. It is already clear that this is a plugin and it's about Flyte. ￼￼

File: plugins/flyte.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  description: |+2
+      kubectl-flyte is an command line tool for flyte that can be used as an extension to kubectl.
```
Comment (ahmetb) [inline plugins/flyte.yaml:10]: This doesn't describe anything to me really. Can you explain a but what Flyte does?

Please read other plugin descriptions that have an unfamiliar project name like this. See how they explain

**Review by ahmetb — CHANGES_REQUESTED** [review]

**Comment by ahmetb** [comment]
@evalsocket ping. 
Also are you someone who works on flyte plugin?
If not, we would much rather have the plugin authors package and offer it here. It wouldn't be appropriate to distribute their plugin here if they won’t be maintaining and releasing updates, setting up release automation etc.

**Comment by evalsocket** [comment]
Yes......We have automation in place for releasing the krew plugin. It is actively maintain and used by the flyte user

**Comment by kumare3** [comment]
@evalsocket thank you for creating this PR. @ahmetb Let me propose the descriptions here and @evalsocket can adapt them?

`Short`: Monitor, Launch and Manage flyte executions using kubectl
`Long`: Kubectl-flyte can be used to manage workflows in a kubernetes cluster executed by FlytePropeller. Users can monitor, visualize executing workflows, delete/terminate executions in bulk, dive into quota utilization for tenants. The tool allows users to launch yaml based Flyte workflows without needing FlyteAdmin etc. For more information about Flyte refer to https://flyte.org

**Comment by ahmetb** [comment]
Overall sounds good. 

Probably can shorten/fix this:

> Monitor, Launch and Manage flyte executions using kubectl

as

> Monitor, launch and manage flyte executions

Similarly, prefixes such as 

> Kubectl-flyte can be used to

are redundant.

File: plugins/flyte.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: flyte
+spec:
+  version: "v0.5.12"
+  homepage: https://github.com/lyft/flyte
+  shortDescription: Monitor, launch and manage flyte executions
+  description: |+2
+      Kubectl-flyte can be used to manage workflows in a kubernetes cluster executed by FlytePropeller. Users can monitor, visualize executing workflows, delete/terminate executions in bulk, dive into quota utilization for tenants. The tool allows users to launch yaml based Flyte workflows without needing FlyteAdmin etc. For more information about Flyte refer to https://flyte.org
```
Comment (corneliusweig) [inline plugins/flyte.yaml:10]: Please wrap at ~80chars so that it prints nicely on wide and narrow terminals.

**Review by corneliusweig — COMMENTED** [review]
Looks good. Thanks for contributing! I wrapped the description more evenly. Can you please incorporate this change into your automation?

/lgtm
/approve

**Pull request merged**